src/configs/nuage/elasticsearch/index.js

```diff
@@ -116,13 +123,56 @@ const getRequestID = function (queryConfiguration, context) {
     if (Object.keys(parameters).length === 0)
         return queryConfiguration.id;
 
-    return queryConfiguration.id + "[" + JSON.stringify(parameters) + "]";
+    return `${queryConfiguration.vizID}-${queryConfiguration.id}[${JSON.stringify(parameters)}]`;
+}
+
+const addSorting = function (queryConfiguration, sort) {
+    let tmpQueryConfiguration = {...queryConfiguration};
+    tmpQueryConfiguration.query.body.sort = {
+        [sort.column]: {
+            order: sort.order
+        }
+    };
+
+    return tmpQueryConfiguration; 
+}
+
+const addSearching = function (queryConfiguration, search) {
+    let tmpQueryConfiguration = _.cloneDeep(queryConfiguration);
+    if (search.length) {
+        objectPath.push(tmpQueryConfiguration, 'query.body.query.bool.must', ESSearchConvertor(search));
+    }
+
+    return tmpQueryConfiguration; 
+}
+
+const getSizePath = function() {
```

Comment:
I think you mean page size here! Let's call it pageSize since pagination is a common terminology